packages/application/src/shell.ts

```diff
@@ -338,6 +338,31 @@ export class LabShell extends Widget implements JupyterFrontEnd.IShell {
       rootLayout.insertWidget(2, this._menuHandler.panel);
     }
 
+    // Skip Links
+    const skipLinkWidget = (this._skipLinkWidget = new Widget());
+    const skipToFilterFiles = document.createElement('a');
+    const toggleSkipLink = (toShow: boolean) => {
+      skipLinkWidget.node.classList.toggle('jp-skiplink--focus', toShow);
+    };
+    skipToFilterFiles.href = '#';
+    skipToFilterFiles.tabIndex = 1;
+    skipToFilterFiles.text = 'Skip to Filter Files';
+    skipToFilterFiles.className = 'skip-link';
+    skipToFilterFiles.addEventListener('focus', e => toggleSkipLink(true));
+    skipToFilterFiles.addEventListener('blur', e => toggleSkipLink(false));
+    skipToFilterFiles.addEventListener('click', e => {
+      const input = document.querySelector(
+        '#filename-searcher .bp3-input'
+      ) as HTMLInputElement;
+      console.log('awesome', input);
```

Comment:
Was this just for debugging? Probably want to remove any extra console logs

Reply:
Oops...Removed.